src/Applications/LDAS_App/GEOSldas_LDAS.rc

```diff
@@ -116,6 +116,18 @@ PERTURBATIONS: 0
 FIRST_ENS_ID: 0
 
 
+# ---- Ensemble forcing
```

Comment:
@saraqzhang, @weiyuan-jiang:
I moved the rc variable up in GEOSldas_LDAS.rc and expanded the comments.  Can you please proofread the comments and let me know if they are ok?

Reply:
yes a  thorough coverage in the expanded comments